### DISCUSSION TITLE
Missing Speed Sensors

### DISCUSSION THREAD

**Discussion by timhill3100** [Rem0o/FanControl.Releases#1553] [Q&A]
Intermittently I get this issues where the GPU fan speed sensors go unfound and my case/CPU fan speeds go all out of wack. The case fans read 3000-9000 rpm and the CPU fans read 20.7 rpm fixed. No idea what is causing it and I have tried everything that makes sense. (deleting all other software that controls fans, redownloading, updating versions, restarting the PC). Any ideas yall have would be amazing. 

(Motherboard is an ASUS B450M-A-2, All system fans are Noctua NF-A12s, GPU is a FE 3060ti)

**Comment by gmr10d** [comment]
I would check out this link, seems to partially help me: https://github.com/LibreHardwareMonitor/LibreHardwareMonitor/issues/614

**Reply by timhill3100** [reply]
I already had HWinfo64 running while i was having these issues

**Reply by gmr10d** [reply]
HWinfo64 won't fix it, the suggestion in the link is for HWMonitor

**Reply by timhill3100** [reply]
I just realized there was a difference between the 2 lol. It seems to fix it. Does it always have to be running to fix the issue? Im wondering what having hwmonitor changes.
